realm/realm-library/src/main/java/io/realm/RealmModel.java

```diff
@@ -0,0 +1,28 @@
+package io.realm;
+
+import io.realm.annotations.RealmClass;
+
+
+/**
+ * Interface for marking classes as RealmObjects, it can be used instead of extending {@link RealmObject}.
+ *
+ * All helper methods available to classes that extend RealmObject are instead available as static methods:
+ *
+ * <pre>
+ * {@code
+ *   Person p = realm.createObject(Person.class);
+ *
+ *   // With the RealmModel interface
+ *   RealmObject.isValid(p);
```

Comment:
No way around this, as that is the trade-off you get with no baseclass. We can always debate where to put the methods though. I see 4 options:

1) Static methods on Realm
2) instance methods on Realm
3) Static methods on RealmObject
4) Somewhere else?

I went with 3) as it avoids polluting the Realm class with to many unrelated methods + it emphasiss the context more to use RealmObject IMO. Happy to hear arguments for other locations though.